### ISSUE TITLE
Default resolution argument for template rasters

### ISSUE THREAD

**Issue by ccarmour** [ninoxconsulting/PEMprepr#37]
In `create_template_raster()` function, default resolution argument is 25m; pipeline step creating template rasters for 5m analysis needs to be more explicitly defined.

Suggestion: adjust the `res` argument to take a vector of integers - e.g., c(5, 25) - and create them with just one run.

**Comment by gcperk** [comment]
@ccarmour yes this is something I was thinking about also, however it gets complex when needing to define two separate location output paths (ie one with the model inputs) and one with the landscape scale sample plan inputs. I will leave this issue open for now and revisit after more testing if needed.

**Comment by gcperk** [comment]
No more issues raised so closing this issue
